GTG/gtk/editor/editor.py

```diff
@@ -79,11 +81,12 @@ def __init__(self,
             "delete_clicked": self.delete_task,
             "on_duedate_pressed": lambda w: self.on_date_pressed(
                 w, GTGCalendar.DATE_KIND_DUE),
+            "on_tags_popover": self.open_tags_popover,
             "on_startdate_pressed": lambda w: self.on_date_pressed(
                 w, GTGCalendar.DATE_KIND_START),
             "on_closeddate_pressed": lambda w: self.on_date_pressed(
                 w, GTGCalendar.DATE_KIND_CLOSED),
-            "close_clicked": self.close,
+            # "close_clicked": self.close,
```

Comment:
Remove this. I can't find the reference to the original close_clicked signal anywhere